azure-pipelines.yml

```diff
@@ -17,7 +17,8 @@ stages:
         timeoutInMinutes: 90
         displayName: 'Validation'
         variables:
-          buildOutputLocation: 'D:\tools'
+          - name: DiffFile
+            value: '$(Build.ArtifactStagingDirectory)\format.diff'

```

Comment:
This repeats `format.diff` below (for `ArtifactName`). Would it be better to centralize that as `DiffFile`, and then make this `DiffPath` (or something similarly named)?

Reply:
I don't personally think so; it's not necessary that these be the same names, it just happens to be that way, so I don't want to tie them together like that.